test/src/main/java/org/apache/accumulo/test/fate/user/FateMutatorImplIT_SimpleSuite.java

```diff
@@ -79,21 +79,26 @@ public void putRepo() throws Exception {
           FateId.from(FateInstanceType.fromNamespaceOrTableName(table), UUID.randomUUID());
 
       // add some repos in order
-      FateMutatorImpl<FateIT.TestEnv> fateMutator = new FateMutatorImpl<>(context, table, fateId);
-      fateMutator.putRepo(100, new FateIT.TestRepo("test")).mutate();
-      FateMutatorImpl<FateIT.TestEnv> fateMutator1 = new FateMutatorImpl<>(context, table, fateId);
-      fateMutator1.putRepo(99, new FateIT.TestRepo("test")).mutate();
-      FateMutatorImpl<FateIT.TestEnv> fateMutator2 = new FateMutatorImpl<>(context, table, fateId);
-      fateMutator2.putRepo(98, new FateIT.TestRepo("test")).mutate();
+      FateMutatorImpl<FateITBase.TestEnv> fateMutator =
+          new FateMutatorImpl<>(context, table, fateId);
+      fateMutator.putRepo(100, new FateITBase.TestRepo("test")).mutate();
+      FateMutatorImpl<FateITBase.TestEnv> fateMutator1 =
+          new FateMutatorImpl<>(context, table, fateId);
```

Comment:
A lot of these could be made more succinct with var:

```suggestion
      var fateMutator1 = new FateMutatorImpl<FateITBase.TestEnv>(context, table, fateId);
```

And even shorter if the import was for `FateITBase.TestEnv`:

```suggestion
      var fateMutator1 = new FateMutatorImpl<TestEnv>(context, table, fateId);
```

Reply:
Addressed in 4c6c31a00bfaf35d94660645b0ca609cf30d8c44